BinUtils/BinUtils/main.swift

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
No, main.swift just contains quick tests like in a playground.

The real tests are in the BinUtilsTests target.